lib/check-self-update/request-promise.js

```diff
@@ -0,0 +1,14 @@
+const request = require('request');
+
+const requestPromise = (url) => {
+  return new Promise((resolve, reject) => {
+    request(url, function(err, data, body){
+      if (err) {
+        reject(err)
```

Comment:
@rafaelcalpena This is going to fail when the user doesn't have an internet connection. We should ensure `lsc` can be used without one.